java/src/jmri/jmrit/throttle/ThrottleBundle_de.properties

```diff
@@ -61,6 +61,7 @@ ButtonProgram = Programmieren
 AddressInUse = Adresse bereits einem anderen Fahrregler zugewiesen.
 
 # ControlPanelPropertyEditor
+ControlPanelProperties =  Eigenschaften
```

Comment:
This should be removed as it is [already defined in line 158](https://github.com/JMRI/JMRI/pull/920/files#diff-15b41ebd2d2a8272a3979664cf4efc44R158)